src/applications/vaos/tests/covid-19-vaccine/components/ConfirmationPage.unit.spec.jsx

```diff
@@ -95,4 +95,111 @@ describe('VAOS vaccine flow <ConfirmationPage>', () => {
       expect(screen.history.location.pathname).to.equal('/');
     });
   });
+
+  it('should verify VA in person calendar ics file format', async () => {
+    const start = moment().tz('America/Denver');
+    const store = createTestStore({
+      covid19Vaccine: {
+        submitStatus: FETCH_STATUS.succeeded,
+        newBooking: {
+          data: {
+            vaFacility: '983',
+            clinicId: '455',
+            date1: [start.format()],
+          },
+          availableSlots: [
+            {
+              start: start.format(),
+              end: start
+                .clone()
+                .add(30, 'minutes')
+                .format(),
+            },
+          ],
+          clinics: {},
+          facilities: [
+            {
+              id: '983',
+              name: 'Cheyenne VA Medical Center',
+              address: {
+                postalCode: '82001-5356',
+                city: 'Cheyenne',
+                state: 'WY',
+                line: ['2360 East Pershing Boulevard'],
+              },
+              telecom: [
+                {
+                  system: 'phone',
+                  value: '307-778-7550',
+                },
+              ],
+            },
+          ],
+        },
+      },
+    });
+
+    const screen = renderWithStoreAndRouter(<ConfirmationPage />, {
+      store,
+    });
+
+    const ics = decodeURIComponent(
+      screen
+        .getByRole('link', {
+          name: `Add ${start.format(
+            'MMMM D, YYYY',
+          )} appointment to your calendar`,
+        })
+        .getAttribute('href')
+        .replace('data:text/calendar;charset=utf-8,', ''),
+    );
+    const tokens = ics.split('\r\n');
+
+    // TODO: Debugging
+    // console.log(tokens);
+
+    expect(tokens[0]).to.equal('BEGIN:VCALENDAR');
+    expect(tokens[1]).to.equal('VERSION:2.0');
+    expect(tokens[2]).to.equal('PRODID:VA');
+    expect(tokens[3]).to.equal('BEGIN:VEVENT');
+    expect(tokens[4]).to.contain('UID:');
+    expect(tokens[5]).to.equal(
+      'SUMMARY:Appointment at Cheyenne VA Medical Center',
```

Comment:
Did we not have any covid specific text for the calendar appointments?

Reply:
I don't think so.